Documentation/platforms/sim/sim/index.rst

```diff
@@ -54,3 +56,36 @@ When the compilation finishes it will create a ``nuttx`` binary, then run it::
     NuttX 10.1.0 508215581f Sep  3 2021 10:47:34 sim sim
     nsh>
 
+Running LVGL
+============
+
+It is possible to run the LVGL Demo directly in the NuttX simulator ::
+
+    $ make distclean
+
+    $ ./tools/configure.sh sim:lvgl
+
+    $ make -j
+
+    $ ./nuttx
+
+Running VNC Server
+==================
+
+NuttX supports a VNC server, so it means even boards without a LCD display
+could export a display interface over network. Also you can test it on NuttX
+simulator before getting it working on your board, just follow these steps ::
+
+    $ make distclean
+
+    $ ./tools/configure.sh sim:vncserver
+
+    $ make -j
+
+    $ ./nuttx
+
+Open a new terminal and execute ::
+
+    $ remmina -c vnc://127.0.0.1
```

Comment:
will `$ remmina -c vnc://localhost` work as well? I'm just thinking about theoretical case when only IPv6 is enabled

Reply:
True, localhost should be a better idea!